cmd/kots/cli/velero.go

```diff
@@ -63,6 +83,299 @@ func EnsurePermissionsCmd() *cobra.Command {
+			viper.BindPFlags(cmd.Flags())
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			v := viper.GetViper()
+
+			namespace := v.GetString("namespace")
+			if err := validateNamespace(namespace); err != nil {
+				return err
+			}
+
+			nfsPath := v.GetString("nfs-path")
+			nfsServer := v.GetString("nfs-server")
+
+			if nfsPath == "" {
+				return errors.New("--nfs-path is required")
+			}
+			if nfsServer == "" {
+				return errors.New("--nfs-server is required")
+			}
+
+			fileSystemConfig := snapshottypes.FileSystemConfig{
+				NFS: &snapshottypes.NFSConfig{
+					Path:   nfsPath,
+					Server: nfsServer,
+				},
+			}
+
+			registryOptions, err := getRegistryConfig(v)
+			if err != nil {
+				return errors.Wrap(err, "failed to get registry config")
+			}
+
+			log := logger.NewCLILogger()
+
+			return veleroConfigureFileSystem(cmd.Context(), namespace, log, fileSystemConfig, registryOptions, v.GetBool("skip-validation"), v.GetString("output"))
+		},
+	}
+
+	cmd.Flags().String("nfs-path", "", "the path that is exported by the NFS server")
+	cmd.Flags().String("nfs-server", "", "the hostname or IP address of the NFS server")
+	cmd.Flags().StringP("namespace", "n", "", "the namespace in which kots/kotsadm is installed")
+	cmd.Flags().Bool("skip-validation", false, "skip the validation of the store configuration")
+	cmd.Flags().StringP("output", "o", "", "Output format. Supported values: json")
+
+	registryFlags(cmd.Flags())
+
+	return cmd
+}
+
+func VeleroConfigureHostPathCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "configure-hostpath",
+		Short:         "Configure snapshots to use a host path as storage destination",
+		Long:          ``,
+		SilenceUsage:  true,
+		SilenceErrors: false,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			viper.BindPFlags(cmd.Flags())
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			v := viper.GetViper()
+
+			namespace := v.GetString("namespace")
+			if err := validateNamespace(namespace); err != nil {
+				return err
+			}
+
+			hostPath := v.GetString("hostpath")
+
+			if hostPath == "" {
+				return errors.New("--hostpath option is required")
+			}
+
+			fileSystemConfig := snapshottypes.FileSystemConfig{
+				HostPath: &hostPath,
+			}
+
+			registryOptions, err := getRegistryConfig(v)
+			if err != nil {
+				return errors.Wrap(err, "failed to get registry config")
+			}
+
+			log := logger.NewCLILogger()
+
+			return veleroConfigureFileSystem(cmd.Context(), namespace, log, fileSystemConfig, registryOptions, v.GetBool("skip-validation"), v.GetString("output"))
+		},
+	}
+
+	cmd.Flags().String("hostpath", "", "a local host path on the node")
+	cmd.Flags().StringP("namespace", "n", "", "the namespace in which kots/kotsadm is installed")
+	cmd.Flags().Bool("skip-validation", false, "skip the validation of the store configuration")
+	cmd.Flags().StringP("output", "o", "", "Output format. Supported values: json")
```

Comment:
this is not new, but something is going on with capitalization here :)

Reply:
i'll fix it